src/widgets/ot_carousel.eliom

```diff
@@ -554,6 +554,7 @@ let%shared make_lazy
   Lwt.wakeup wakener carousel.elt; (* generate initial content (only client-side) *)
   (* replace spinners with content when switched to for the first time *)
   let _ = [%client (
+    if ~%spinners_and_generators = [] then () else
```

Comment:
The test could be put server-side and we could change it to checking whether the length of the list is strictly greater than 1. But this is good enough.